### PR TITLE
rename serie to chain

### PR DESCRIPTION
**Pull request by flokli**
This seems to be a less confusing, as it's already used in other places
using gerrit.

cc @sternenseemann

### CODE REVIEW

File: gerrit/chain.go [hunk 1]
```diff
@@ -65,20 +65,20 @@ func (s *Serie) CheckIntegrity() error {
...
 }
 
-// FilterAllChangesets applies a filter function on all of the changesets in the series.
+// AllChangesets applies a filter function on all of the changesets in the chain.
 // returns true if it returns true for all changesets, false otherwise
-func (s *Serie) FilterAllChangesets(f func(c *Changeset) bool) bool {
+func (s *Chain) AllChangesets(f func(c *Changeset) bool) bool {
 	for _, changeset := range s.ChangeSets {
-		if f(changeset) == false {
+		if f(changeset) {
```
Comment (sternenseemann) [inline gerrit/chain.go]: This inverts the logic and it doesn't seem like there's a change at the call site?

Comment (flokli) [reply gerrit/chain.go]: Thanks! Seems my keyboard got stuck and I didn't add the `!` when simplifying the expression. Will fix.

Comment (flokli) [reply gerrit/chain.go]: Fixed.

**Comment by tazjin** [comment]
> as it's already used in other places

Additionally, it's a real word :)

LGTM

**Pull request merged**